config_tors.yml

```diff
@@ -1,11 +1,8 @@
 ---
-
 - name: config tors
   hosts: localhost
-  gather_facts: true
-
   vars:
-    - config_dir: /var
```

Comment:
I still struggle with this myself: when is something a list and when is it not? In this case it is not, because the variable is directly referenced. My (very imperfect) mnemonic device for this is: only if you can/would loop over the items, it is a list.